frontend/src/hooks/useSingleToArray.test.js

```diff
@@ -0,0 +1,35 @@
+import { renderHook } from '@testing-library/react-hooks';
+import useSingleToArray from './useSingleToArray';
+
+describe('useSingleToArray', () => {
```

Comment:
I think the `useSingleToArray` function may actually be superficial now, as we enforce sections being sent as an array on the backend.

Reply:
I checked and it doesn't seem to be used anymore in the whole frontend application and shall therefore remove it and its tests from the codebase.

<img width="437" alt="afbeelding" src="https://github.com/Amsterdam-Music-Lab/MUSCLE/assets/8208970/3e3bb4d9-f7f5-4c89-b6f2-77e79e91aced">